infra/cifuzz/cifuzz_test.py

```diff
@@ -97,6 +98,28 @@ def command_has_env_var_arg(command, env_var_arg):
 class BuildFuzzersIntegrationTest(unittest.TestCase):
   """Integration tests for build_fuzzers."""
 
+  def setUp(self):
+    test_helpers.patch_environ(self)
+
+  def test_external_project(self):
+    """Tests building fuzzers from an external project."""
+    project_name = 'external-project'
+    project_src_path = os.path.join(TEST_FILES_PATH, project_name)
+    build_integration_path = os.path.join(project_src_path, 'oss-fuzz')
+    os.environ['PROJECT_SRC_PATH'] = project_src_path
```

Comment:
Can some of this init move into setUp

Reply:
I don't think there's a point to putting something in setUp unless it is used in multiple tests.
Do you have any particular suggestions?